gateway/1.9/modules/ROOT/pages/flex-conn-replica-caching.adoc

```diff
@@ -0,0 +1,24 @@
+= Configuring Flex Gateway Caching in Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Enable Flex Gateway configuration caching to have a Flex Gateway cache its gateway configuration in Redis shared storage. When configured, Flex replicas use the cached gateway configuration instead of each replica directly downloading the configuration from Anypoint Platform. Caching configuration reduces scale-up time and increases resiliency.
```

Comment:
```suggestion
Enable Flex Gateway configuration caching to have a Flex Gateway cache its gateway configuration in Redis shared storage. Flex replicas then use the cached gateway configuration instead of each replica directly downloading the configuration from Anypoint Platform. Caching configuration reduces scale-up time and increases resiliency.
```

Reply:
Use When for time